tests/Database/ScriptsTest.php

```diff
@@ -5,7 +5,7 @@
 use Sokil\IsoCodes\IsoCodesFactory;
 use Sokil\IsoCodes\Database\Scripts\Script;
 
-class ScriptsTest extends \PHPUnit_Framework_TestCase
+class ScriptsTest extends \PHPUnit\Framework\TestCase
```

Comment:
May I ask you a question? Why did you do this?)))

Reply:
To upgrade the ```PHPUnit``` version, then setting the current PHPUnit namespace for the newer PHPUnit version.

Reply:
I mean why you do this in all this libs? It's just tests))

Reply:
I think using the stable PHPUnit version is better than using older.